src/main/java/com/iri/dao3/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,118 @@
+package com.iri.dao3.dao.jdbc;
+
+import com.iri.dao3.dao.ManufacturerDao;
+import com.iri.dao3.exception.DataProcessingException;
+import com.iri.dao3.lib.Dao;
+import com.iri.dao3.model.Manufacturer;
+import com.iri.dao3.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertQuery = "INSERT INTO manufacturer (name, country) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement =
+                        connection.prepareStatement(insertQuery, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            int result = preparedStatement.executeUpdate();
```

Comment:
```suggestion
           preparedStatement.executeUpdate();
```
we don't need it, if we can't add manufacturer, we will receive SQL exception

Reply:
rewote